acceptance/openstack/containerinfra/v1/containerinfra.go

```diff
@@ -177,3 +179,87 @@ func WaitForCluster(client *gophercloud.ServiceClient, clusterID string, status
 		return false, nil
 	})
 }
+
+// CreateProject will create a project with a random name.
+// It takes an optional createOpts parameter since creating a project
+// has so many options. An error will be returned if the project was
+// unable to be created.
+func CreateProject(t *testing.T, client *gophercloud.ServiceClient, c *projects.CreateOpts) (*projects.Project, error) {
```

Comment:
Instead of recreating this function, you can import it from the acceptance tests. For example:

https://github.com/gophercloud/gophercloud/blob/master/acceptance/openstack/networking/v2/extensions/layer3/routers_test.go#L9

https://github.com/gophercloud/gophercloud/blob/master/acceptance/openstack/networking/v2/extensions/layer3/routers_test.go#L20

So here it would be:

```go
import (
  identity "github.com/gophercloud/gophercloud/acceptance/openstack/identity/v3"
)

project, err := identity.CreateProject(t, client, nil)
```